docs/integrate_ultra.md

```diff
@@ -0,0 +1,40 @@
+# Integration with Ultra
+
+This section will guide you while you are integrating your application with Ultra. If you want to have a visual walk-through before following the detailed instructions, please head over to our [Ultra Demo](https://docs.google.com/document/d/1o3j1QRr4PofuJvWaFhsAN18vnLURFMEMBKpqD09Dt2Y/edit#heading=h.x9yfo0xokx3e) segment. You can open the Ultra playground application on your browser there and learn about implementation of the APIs.
+
+Before we move on to the steps of integration, we want you to understand the high-level picture of the data-flow between different systems. The user’s data will flow to the partner’s end through Ultra. At user’s side, the internal APIs will interact with the partner’s UI within Ultra container and the user’s information such as login details will pass through the Flipkart’s ecosystem to the partner’s side with the help of the Ultra APIs. At partner’s side, the application will interact with the internal APIs of Flipkart and Ultra to gain user data and open access to their application. Following is the diagrammatic representation of this high level scenario:
```

Comment:
We sometimes refer to the reader as our partner, and sometimes we use the term "partner" explicitly. Should we standardize it ? I feel referring to the reader by always saying "you" might be better, What say ?

Reply:
I think we should standardise it as this can be referred by anyone from a developer to a product manager. If we use explicitly the term "partner", it can appear in different terms to some people. Using "you" appeals to the reader irrespective of their backgrounds.